DayrSuryan/DSEthiop10.xml

```diff
@@ -481,10 +488,23 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                </physDesc>
 
                <history>
-               <!--   <origin>
-                  </origin>-->
+                   <origin>
+                <origDate notBefore="1930" notAfter="1960">From 1930 to the mid-20th century.</origDate>
```

Comment:
The text written in origDate is the one that will appear in the summary line in the PDF. Maybe a more concise expression would be better? "1930 - mid-20th century"?

Reply:
Ok.